### PR TITLE
Create define for top toolbar clearance and fix #837

### LINKED ISSUE #837: Can't move windows on title screen to where the toolbar would be
Look at:

Toolbar bug

The scenerio selector can't be dragged any higher than that, because it is "blocked" by the toolbar (which of course doesn't exist on the title screen).

### PR DESCRIPTION
**Pull request by Gymnasiast**

### CODE REVIEW

File: src/openrct2/interface/window.h [hunk 1]
```diff
@@ -36,6 +36,7 @@ struct TitleSequence;
 struct TextInputSession;
 
 #define TEXT_INPUT_SIZE 1024
+#define TOP_TOOLBAR_CLEARANCE 28
```
Comment (marijnvdwerf) [inline src/openrct2/interface/window.h]: What's wrong with `HEIGHT`?

Comment (Gymnasiast) [inline src/openrct2/interface/window.h]: The height is 27. The clearance is 28.

Comment (marijnvdwerf) [inline src/openrct2/interface/window.h]: I've never heard of 'Clearance' in UI design. 'Keyline' maybe, but height seems like the most fitting term.

Comment (Gymnasiast) [inline src/openrct2/interface/window.h]: Feel free to create a define for TOP_TOOLBAR_HEIGHT that has the value of 27 and change the existing usages of TOP_TOOLBAR_CLEARANCE to TOP_TOOLBAR_HEIGHT + 1.

Comment (IntelOrca) [inline src/openrct2/interface/window.h]: Maybe `TOP_TOOLBAR_BOTTOM` ?

Comment (Gymnasiast) [inline src/openrct2/interface/window.h]: The bottom of the toolbar is at 27, strictly speaking.

*Sigh* Why is it always small stuff like this that gets so much discussion? Marijn's comment is a good example of getting bogged down. This makes creating PRs to clean up stuff a pain in the arse, to be quite honest.

Comment (IntelOrca) [inline src/openrct2/interface/window.h]: At least with the `< 28` you can make that `<= TOP_TOOLBAR_HEIGHT`

Comment (Gymnasiast) [inline src/openrct2/interface/window.h]: Ugh, fine, I'll create a define for the height (27) and adjust the `<` accordingly.

**Comment by Gymnasiast** [comment]
I changed TOP_TOOLBAR_CLEARANCE to TOP_TOOLBAR_HEIGHT and adjusted its uses accordingly.

**Pull request merged**